yucca/umbrel-app.yml

```diff
@@ -0,0 +1,30 @@
+manifestVersion: 1.1
+id: yucca
+category: automation
+name: Yucca
+version: "0.10.0"
+tagline: Simple NVR solution video surveillance
+description: >-
+  Simple NVR solution video surveillance for home and business.  
+  Perfect for both a solution for safe cities with thousands of cameras and launching a subscriber service, as well as for home use self-hosted.
+
+releaseNotes: >-
+  Init Yucca on Umbrel
+
+  Full release notes are available at https://docs.yucca.app/en/release_notes/0.10.0/
```

Comment:
```suggestion
```

Since this is the initial release of the app on Umbrel, there release notes should probably be removed.